backend/docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
   api:
     build:
       context: .
+      dockerfile: Dockerfile.prod
```

Comment:
If we're not using the main Dockerfile any more we could just call Dockerfile.prod Dockerfile—but do we actually want to be running Insights on front-end dev machines?